### DISCUSSION TITLE
Button escape

### DISCUSSION THREAD

**Discussion by ALTITUDE-DEV-FR** [GrapesJS/grapesjs#5111] [Q&A]
Hello, I have a slight issue that I can't seem to understand. When I'm editing a button (btn), I'm unable to press the space key on my keyboard to create a button that looks like this:

MY SUPER BUTTON

For example, if I want to go back to:

MYSUPERBUTTON it works, but if I want to add spaces in the button, it's impossible for me to create "MY SUPER BUTTON."

How can I fix this? Thank you for this amazing tool! Thanks for your help.

**Comment by ALTITUDE-DEV-FR** [comment]
https://i.imgur.com/v0rsbU2.jpg

**Comment by artf** [comment]
Some DOM elements have a few limitations in inline text editing and button is one of those.
You can see [here](https://github.com/GrapesJS/components-forms/blob/6a63e9edb7b085790e90e98acf54accc5f4bd642/src/components.ts#L260-L304) the button implementation from the form plugin where it was easier to move the text editing as a Trait instead of inline editing, this allows to prevent these kind of limitations.
